docs/hazmat/primitives/asymmetric/serialization.rst

```diff
@@ -379,6 +457,18 @@ Serialization Formats
         The public key format used by OpenSSH (e.g. as found in
         ``~/.ssh/id_rsa.pub`` or ``~/.ssh/authorized_keys``).
 
+.. class:: ParameterFormat
+
+    .. versionadded:: 1.9
+
+    An enumeration for parameters formats. Used with the ``parameter_bytes``
+    method available on
+    :class:`~cryptography.hazmat.primitives.asymmetric.dh.DHParametersWithSerialization`.
+
+    .. attribute:: ASN1
+
+        Abstract Syntax Notation One.
```

Comment:
I am not real sure what you mean here. The output of the asn1 parser is:

```sh
openssl asn1parse -inform DER -in vectors/cryptography_vectors/asymmetric/DH/dhp.der 
    0:d=0  hl=3 l= 135 cons: SEQUENCE          
    3:d=1  hl=3 l= 129 prim: INTEGER           :BB2B5ADE38535A23A93CBCF28D307E3CAC044E7557A56D6AB8E6DBEE41B2540AFD1B659DD1DD7B7831F9C70DB3E22700DB3372E24523C007B78EF8DE4191CF2E574A5BA5E2AE1D0C910D387A32A98C6E785B8554910B1302D05EB5FF31C2B0FA14BA18F5509F0EAD661806A3D6A974D75881E2D079075E77C06E9BC8EACCFDEB
  135:d=1  hl=2 l=   1 prim: INTEGER           :02
```
This is just a sequence of the p,g